tests/mssql_test.js

```diff
@@ -0,0 +1,24 @@
+import sql from 'k6/x/sql';
+
+const db = sql.open("sqlserver", "./test.db");
```

Comment:
The second argument should be a connection string to showcase how it's specified for MS SQL, but it's OK, I'll fix it when I add the CI tests.